packages/plugins/rubygem-smart-proxy-probing/rubygem-smart-proxy-probing.spec

```diff
@@ -1,44 +1,51 @@
-# Generated from smart-proxy-probing-0.0.2.gem by gem2rpm -*- rpm-spec -*-
 # template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name smart-proxy-probing
 %global plugin_name smart-proxy-probing
 
-%global foreman_proxy_min_version 1.17
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
-%global smart_proxy_dynflow_bundlerd_dir %{?rhel:/opt/theforeman/tfm/root/}%{_datadir}/smart_proxy_dynflow_core/bundler.d
+%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{!?scl:/opt/theforeman/tfm/root}%{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
`%{!?scl:/opt/theforeman/tfm/root}` Isn't there an scl macro that evaluates to /opt/theforeman/tfm/root ? seems like this should be builtin to the scl system.

Reply:
yup, `%_scl_root`[0].  Is there a reason it's not being used here?

[0]https://www.softwarecollections.org/en/docs/guide/#sect-Software_Collection_Macros

Reply:
ah, nevermind, i read the comment below, "not" being important here.  Ignore me.